CHANGELOG.md

```diff
@@ -2,9 +2,15 @@
 
 Embedded Graphics is a `no_std` library for adding graphics features to display drivers. It aims to use the minimum amount of memory for builtin graphics objects by leveraging Rust's iterators to avoid large allocations. It targets embedded environments, but can run anywhere like a Raspberry Pi up to full desktop machines.
 
+## Unreleased
+
+### Added
+
+- Added a more complex `clock` example to the simulator - [check it out](https://github.com/jamwaffles/embedded-graphics/tree/embedded-graphics-v0.6.0-alpha.3/simulator/examples/clock.rs) for some more in-depth usage of Embedded Graphics.
```

Comment:
Could you rename the example to `analog_clock` or something similar? I have an idea for a way to draw seven segment displays and would probably demonstrate that with another clock example.